language/tests/regent/compile_fail/optimize_index_launch_noninjective.rg

```diff
@@ -0,0 +1,34 @@
+-- Copyright 2021 Stanford University
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- fails-with:
+-- Errors reported during runtime.
```

Comment:
This seems to get automatically printed by `std.assert_error`, so I'm not sure how to get rid of it.

Reply:
It's ok for the lines listed here to be a subset of what gets printed. What the test runner actually checks for is that each line listed appears at least once in the output (in any order). (Because you can run on multiple nodes and weird things can happen with the output, so we don't stress about it too much.)

Reply:
I see, fixed.